include/mme-app/utils/mmeTimerTypes.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020-present Infosys Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef INCLUDE_MME_APP_UTILS_MMETIMERTYPES_H_
+#define INCLUDE_MME_APP_UTILS_MMETIMERTYPES_H_
+
+#include <stdint.h>
+#include <timerQueue.h>
+
+namespace mme
+{
+/******************************************************
+ * Timer ID range used by different modules in mme-app
+ ******************************************************/
+const uint16_t stateGuardTimer_c = 1;
+
+/******************************************************
+ * Timeout durations in milliseconds
+ ******************************************************/
+const uint32_t defaultStateGuardTimerDuration_c = 5000;
+
+
+class MmeTimerContext : public TimerContext
+{
+public:
+    MmeTimerContext(uint32_t cbIdx,
+            uint16_t timerType,
+            uint16_t timerId,
+            CTime& expiryTime):
+        TimerContext(expiryTime),
+        controlBlockIdx_m(cbIdx),
+        timerType_m(timerType),
+        timerId_m(timerId)
+    {
+    }
+
+    ~MmeTimerContext()
+    {
+    }
+
+    uint16_t getTimerType() const
+    {
+        return timerType_m;
+    }
+
+    uint16_t getTimerId() const
+    {
+        return timerId_m;
+    }
+
+    uint32_t getCbIndex() const
+    {
+        return controlBlockIdx_m;
+    }
+
+private:
+    uint32_t controlBlockIdx_m;
```

Comment:
Each timer type in MME can define its own timer Class by inheriting public TimerContext. 
Dont keep the name Class MmeTimerContext, Since in this case you are running timer for control block.
So keep name like, class  mmeControlBlockTimer::public TimerContext

Reply:
Updated accordingly. Renamed to MmeUeTimerContext..